docs/getting-started/ios/fastlane-swift.md

```diff
@@ -42,6 +42,59 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminary feature that allows the usage of _fastlane_ with other SPM packages.
+
+### Step 1
+
+Create an executable Swift Package Manager project with `swift package init --type executable`.
+
+### Step 2
+
+Add the _fastlane_ dependency to your `Package.swift`.
+
+```swift
+.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", .from("2.179.0"))
+```
+
+### Step 3
+
+Create your Fastfile.swift file in your package and add the desired lanes, as follows.
+
+``` swift
+import Fastlane
+
+// Create a class with: 
+class FastFile: LaneFile {
+    // Your lanes goes here.
+}
+```
+
+### Step 4
+
+Add an entry point (`@main`) or a `main.swift` file (mandatory for executable SPM packages) and don't forget to start the _fastlane_ runloop as follows:
+
+```swift
+import Fastlane
+
+Main().run(with: Fastfile())
+```
+
+### Step 5
+
+Modify the target of your executable to have executable arguments `lane myLane` or add them in the call after making `swift build`.
```

Comment:
With me still being rather super new to all of this, is there a tutorial elsewhere that might explain how to do this? 😅

Reply:
If you generate an Xcode project, you can change the arguments being passed into the executable in the target options or just add them into the generated executable as if it was other CLI.